pkg/volume/fc/disk_manager.go

```diff
@@ -57,7 +57,31 @@ func diskSetUp(manager diskManager, b fcDiskMounter, volPath string, mounter mou
 	}
 	err = mounter.Mount(globalPDPath, volPath, "", options)
 	if err != nil {
-		glog.Errorf("failed to bind mount:%s", globalPDPath)
+		glog.V(4).Infof("Mount failed: %v", err)
+		noMnt, err = b.mounter.IsLikelyNotMountPoint(volPath)
+		if err != nil {
+			glog.Errorf("IsLikelyNotMountPoint check failed: %v", err)
+			return err
+		}
+		if !noMnt {
+			if err = b.mounter.Unmount(volPath); err != nil {
```

Comment:
This should use mntERR instead of err.
if mntERR = b.mounter.Unmount(volPath); mntERR != nil {